OREData/ored/portfolio/fxdigitaloption.cpp

```diff
@@ -51,6 +51,7 @@ void FxDigitalOption::build(const boost::shared_ptr<EngineFactory>& engineFactor
     bool flipResults = false;
     if (payoffCurrency_ == "") {
         DLOG("PayoffCurrency defaulting to " << domesticCurrency_ << " for FxDigitalOption " << id());
+        payoffCurrency_ = domesticCurrency_;
```

Comment:
I do not see any advantage of populating the field, why do you want to do this?

A disadvantage is that it toXML(fromXML(x)) != x.